src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -74,13 +74,10 @@ public function load($resource, $type = null)
 
         try {
             $collection = parent::load($resource, $type);
-        } catch (\Exception $e) {
+        } finally {
```

Comment:
@nicolas-grekas I assume this hack also works with finally, i.e. fatal errors are skipping it. I tried to test it as described in #14342. And it had the same result. So this code change didn't change the outcome of the exception page. But it was kinda strange anyway as there was a second error below the real one:

```
FatalErrorException in TraceableEventDispatcher.php line 357:
Error: Cannot use object of type Symfony\Component\EventDispatcher\Debug\WrappedListener as array

in vendor\symfony\symfony\src\Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher.php at line 357
```